docs/guides/external-node/docker-compose-examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Running External node locally
+
+This directory contains two docker compose files for running external nodes, for sepolia testnet and mainnet.
+
+They are prepared to work out-of-the-box, no config changes are needed
+
+By default, the HTTP JSON-RPC API will run on port `3050`, while WS API will run on port `3051`.
```

Comment:
Bikeshedding: IMO, placing each sentence on a separate paragraph here is an overkill; it'd read better if all 3 sentences were on the same paragraph. Ditto with two sentences in the note below.

Reply:
I've rewritten this document into 00_quick_start, there are still some one-sentence paragraphs left, but imo they make sense